src/router_core/connections.c

```diff
@@ -1900,7 +1900,7 @@ static void qdr_link_inbound_second_attach_CT(qdr_core_t *core, qdr_action_t *ac
         // Handle incoming link cases
         //
         switch (link->link_type) {
-        case QD_LINK_ENDPOINT:
+        case QD_LINK_ENDPOINT: {
```

Comment:
Writing body of a `case` into a block is IMO good practice. It's not necessary in C, however.